website/source/docs/agent/options.html.markdown

```diff
@@ -546,7 +547,7 @@ definitions support being updated during a reload.
   connections make use of TLS and that the client provides a certificate signed
   by the Certificate Authority from the [`ca_file`](#ca_file). By default, this is false, and
   Consul will not enforce the use of TLS or verify a client's authenticity. This
-  only applies to Consul servers since a client never has an incoming connection.
+  applies to both server RPC and to the HTTPS API.
```

Comment:
I think this should still clearly indicate that it only affects server nodes. Consider

> This only applies to Consul servers, since a client never has an incoming connection. It affects both the RPC and HTTPS interfaces.